README.md

```diff
@@ -200,13 +202,14 @@ Many C++20 features could be considered truly revolutionary, changing old good C
 * [Exploring C++20: The Programmer's Introduction to C++](https://www.amazon.com/dp/1484259602) (Ray Lischner, 2020)
 * [The C++ Standard Library](https://leanpub.com/cpplibrary) (Rainer Grimm, 2021)
 * [Programming with C++20](https://leanpub.com/programming-with-cpp20) (Andreas Fertig, 2021)
-* [C++20 - The Complete Guide](https://leanpub.com/cpp20) (Nicolai M. Josuttis, the book is 90% complete as of May 2022)
```

Comment:
This book 100% complete now.

Reply:
PS. of course not even a single book published by LeanPub is truely 100% complete as authors constantly update their content.